frontend/src/actions/parcelsActions.ts

```diff
@@ -242,3 +243,20 @@ export const storeBuildingDetail = (
     position,
   },
 });
+
+export interface IStoreAssociatedBuildingDetail {
```

Comment:
decided to go with a new field in redux for this as reusing the parcelDetail field causes undesirable side effects.